src/core/autosave.cpp

```diff
@@ -38,10 +41,9 @@ AutosavePrivate::AutosavePrivate(Autosave& document)
 	settingsChanged();
 }
 
-AutosavePrivate::~AutosavePrivate()
-{
-	// nothing, not inlined
-}
+AutosavePrivate::~AutosavePrivate() = default;
```

Comment:
Usually I include such changes when they are not dangerous and when they catch my eye during nearby changes. Since nobody else takes the time to cleanup/beautify/modernize, these changes might never happen otherwise. But I'm okay with separating it out here.